### DISCUSSION TITLE
Github App equivalent for this action

### DISCUSSION THREAD

**Discussion by hkadakia** [github-community-projects/evergreen#223] [Q&A]
Is there an example or documentation on how to use this action as a Github App? I see in the documentation on using Github App ID & other Github App details but having accurate documentation on integrating it with Github App would be helpful.

tl;dr We have multiple orgs and adding a repo for each org & using a PAT may not be a scalable option. One option we are looking at is creating a Github App which does not require PATs and easy to enforce/deploy across multiple orgs.

**Comment by zkoppert** [comment]
This action is currently not set up to run as a GitHub App. Would you want to self host for that? We could look into doing this but a hosted version of the app is unlikely.

**Reply by hkadakia** [reply]
We are open to running it as a self hosted app. If there are any documentation around how to host that and use the app token instead of PAT that would be ideal.
